modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -0,0 +1,251 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="verify-application-pod-access-and-fpga-usagage-on-openess_{context}"]
+= Verifying application pod access and FPGA usage on OpenNESS
+
+OpenNESS is an edge computing software toolkit that enables edge platforms to onboard and manage applications and network functions across any type of network.
+
+To verify that all OpenNESS features are working together, including SR-IOV binding, the device plugin, Wireless Base Band Device (bbdev) configuration, and SR-IOV (FEC) VF functionality inside a non-root pod, you can build an image and run a simple validation application for the device.
+
+For more information, go to link:http://www.openness.org[openess.org].
+
+.Prerequisites
+
+* Intel® FPGA PAC N3000 card (Optional)
+* Node or nodes installed with the n3000-operator
+* Node or nodes installed with the SR-IOV-FEC operator
+* Real-Time kernel and huge pages configured with Performance Addon Operator
+
+.Procedure
+
+. Create a namespace for the test by completing the following actions:
+
+.. Define the `test-bbdev` namespace by creating a file named `test-bbdev-namespace.yaml` file as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-bbdev
+  labels:
+    openshift.io/run-level: "1"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$  oc create -f `test-bbdev-namespace.yaml`
+----
+
+. Create the following `Pod` specification, and then save the YAML in the `pod-test.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-bbdev-sample-app
+  namespace: test-bbdev <1>
+spec:
+  containers:
+  - securityContext:
+      privileged: false
+      name: bbdev-sample-app
+      image: bbdev-sample-app:1.0 <2>
+      command: [ "sudo", "/bin/bash", "-c", "--" ]
+      runAsUser: 0 <3>
+      capabilities:
+        add:
+          - IPC_LOCK
+          - SYS_RESOURCE
+    resources:
```

Comment:
indentation issue should move to the `capabilities` level

Reply:
Hi @SchSeba I was checking against https://github.com/open-ness/openshift-operator/blob/main/spec/openshift-sriov-fec-operator.md and it looks like resources is on the same level as securityContext. You want me to put at the same level as capabilities  and indent others accordingly - correct?

capabilities:
        add:
          - IPC_LOCK
          - SYS_RESOURCE
      resources:
       requests:
         hugepages-1Gi: 4Gi <4>
         memory: 1Gi
         cpu: "4" <5>
         intel.com/intel_fec_5g: '1' <6>
         #intel.com/intel_fec_acc100: '1'
         #intel.com/intel_fec_lte: '1'
       limits:
         memory: 4Gi
         cpu: "4"
         hugepages-1Gi: 4Gi
         intel.com/intel_fec_5g: '1'
         #intel.com/intel_fec_acc100: '1'
         #intel.com/intel_fec_lte: '1'